### DISCUSSION TITLE
Versioning

### DISCUSSION THREAD

**Discussion by daniel-mezo** [webauthn4j/webauthn4j#609] [Q&A] [answered]
Hi @ynojima,

Just wondering how do you plan to version the library. Currently 0.18.0 is released, but it seems to me that the library is rather complete, from a functional POV. Is this a wrong assessment? What do you think is missing for a 1.0.0 release?

Cheers,
Daniel

**Comment by ynojima** [comment, marked as answer]
From a functional POV, yes, it is enough to decrare v1.0.0, but the API isn't stable enough yet, and there have been some (minor but still) breaking changes in the last few months.
https://github.com/webauthn4j/webauthn4j/releases

I would like to declare v1.0.0 if there is no need to make breaking changes for about 4 months.
